.github/workflows/main_push.yml

```diff
@@ -1,17 +1,11 @@
 name: CI
```

Comment:
Let's rename the workflow to have a unique name, please

Reply:
https://github.com/Activiti/Activiti/pull/4529/commits/d5f96261875a6604378c59939a4d890b91061612